python/pyspark/context.py

```diff
@@ -221,7 +246,7 @@ def _do_init(
         if environment:
             for key, value in environment.items():
                 self._conf.setExecutorEnv(key, value)
-        for key, value in DEFAULT_CONFIGS.items():
+        for key, value in DEFAULT_CONFIGS.items():  # type: ignore[assignment]
```

Comment:
Instead of ignoring here, we can add 

```str
DEFAULT_CONFIGS: Dict[str, Any] = {
```

and, for consistency, adjust `environment` in `__init__` and `_do__init` as well.